PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py

```diff
@@ -32,68 +33,58 @@ class AutomatedVoltageUpgrade(AbstractPostprocess):
-        super(AutomatedVoltageUpgrade).__init__()
-        self.__dssinstance = dssInstance
-        New_settings = {
-            "Feeder": "../Test_Feeder_RIN_69_08_2030_sb100",
-            "img_path": "../Images",
-            "DPV_scenarios": "../ten_random_RIN_69_08_2030_sb100",
-            "master file": "MasterDisco.dss",
-            "DPV_penetration_HClimit": 0,
-            "DPV_penetration_target": 200,
-            "DPV control": "PF=1",  # "PF=1" or "PF=-0.95" or "VVar-CatA" or "VVar-CatB" or "VVar-VWatt-CatB"
-            "DPV system priority": "watt",  # "watt" or "var"
-            "Outputs": "C:\Documents_NREL\Grid_Cost_DER_PhaseII\Control_device_placement\Outputs",
-            "V_upper_lim": 1.05,
-            "V_lower_lim": 0.95,
-            "Target_V": 1,
-            "plot window open time": 1,  # seconds
-            "Min PVLoad multiplier": 1,
-            "Min Load multiplier": 0.1,
-            "Max Load multiplier": 1,
-            "Range B upper": 1.05,
-            "Range B lower": 0.95,
-            "nominal_voltage": 120,
-            "nominal pu voltage": 1,
-            "tps_to_test": [0.2, 1.2, 0.1, 0.9],
-            # [min load multiplier without PV, max load multiplier without PV, min load multiplier with PV, max load multiplier with PV]
-            "create topology plots": True,
-            # Set this to true only if coordinates for all buses in the network are available
-            "Cap sweep voltage gap": 1,
-            # This value determines difference increase between cap ON and OFF voltage setting, example (119.5, 120.5), (119,121) and so on
-            "max control iterations": 50,  # max OpenDSS Control iteration limit
-            "reg control bands": [1, 2],  # Reg control voltage bands within which taps will not change
-            "reg v delta": 0.5,  # Reg control voltage set point is varied in this range
-            "Max regulators": 4,
-            # This number gives the maximum number of regulators placed in the feeder apart from substation LTC
-            "Use LTC placement": True,
-        }
-        for key,val in New_settings.items():
-            if key not in self.Settings:
-                self.Settings[key] = val
-        self.logger = Logger
+        super(AutomatedVoltageUpgrade, self).__init__(project, scenario, inputs, dssInstance, dssSolver, dssObjects, dssObjectsByClass, simulationSettings, Logger)
+
+        # TODO Akshay: this seems prone to error. Do we need a programmatic way of getting the filename?
+        thermal_filename = "Thermal_upgrades_pen_0.dss"
+        thermal_dss_file = os.path.join(
+            project.get_post_process_directory(self.config["Thermal scenario name"]),
+            thermal_filename
+        )
+        if not os.path.exists(thermal_dss_file):
+            raise InvalidParameter(f"AutomatedThermalUpgrade did not produce thermal_filename")
+        # TODO Akshay: redirect to the above file where appropriate
```

Comment:
@akshaykj92 Note the two comments here.  You're welcome to push code to this branch or wait until we merge to the develop branch.